sender/sender.cpp

```diff
@@ -0,0 +1,23 @@
+
+#include <iostream>
+#include "../sensor/temperature.h"
+#include "../sensor/state_of_charge.h"
+#include <vector>
+#include "sender.h"
+
+using namespace std;
+
+bool sendSensorDataToConsole(){
+  bool isDataSendSuccessfully = false;
```

Comment:
better we can start name with b(since it is boolean)

Reply:
as we discussed we are following this naming standard in c,c++